src/bindings/js/node/lib/addon.ts

```diff
@@ -60,7 +60,7 @@ interface Model {
   input(nameOrId?: string | number): Output;
   getName(): string;
   isDynamic(): boolean;
-  getOutputShape(): number;
+  getOutputShape(): SupportedTypedArray;
```

Comment:
SupportedTypedArray includes arrays with float typed elements. But shape it's set of numbers having integer value. Current type could be confusing.